chain/client/tests/cross_shard_tx.rs

```diff
@@ -420,9 +420,9 @@ mod tests {
                 block_production_time,
                 drop_chunks,
                 !test_doomslug,
-                20,
+                5,
                 test_doomslug,
-                vec![true; validators.iter().map(|x| x.len()).sum()],
+                vec![false; validators.iter().map(|x| x.len()).sum()],
```

Comment:
why this change? It actually changed the semantics of this test.

Reply:
it is a different change, revert it